src/AnimalNoise.java

```diff
@@ -1,36 +1,37 @@
-
-//change in browser 
-import model.*;
-
-public class AnimalNoise {
-	//add your animal class to the model package
-	//only push this file and the animal class you created
-	
-	// Test
-
-	public static void main(String[] args) {
-		Cat fluffy = new Cat();
-		System.out.println(fluffy.speak());
-    
-		Donkey joe = new Donkey("Joe", "Brown", 4);
-		System.out.println(joe.speak());
-
-		Pig mrCuddles = new Pig("Mr. Cuddles", "pink", 4);
-		System.out.println(mrCuddles.speak());
-		
-		Lion simba = new Lion(false, 2, "Simba");
```

Comment:
Is this all red because these were added before I did a pull request or something else? Do I have to restart this entire process?

Reply:
It was red because the lines did not match what was there before.  When it was edited, a new comment was added that pushed everything down a row.